general-concepts/dependencies/text.xml

```diff
@@ -32,10 +76,9 @@ exemptions).
 <p>
 The <c>RDEPEND</c> ebuild variable should specify any dependencies which are
 required at runtime. This includes libraries (when dynamically linked), any data
-packages and (for interpreted languages) the relevant interpreter. In EAPI=3 or
-older, if this variable is not specified it defaults to the value of
-<c>DEPEND</c>, however the implicit usage is frowned upon. In EAPI=4, the
-implicit behaviour was removed and the assignment is always explicit.
+packages and (for interpreted languages) the relevant interpreter. While
+historically this variable could have implicit value, nowadays it must be
```

Comment:
"an implicit value"?
Does it mean that we need to always specify `DEPEND=""` now?

Reply:
@sbraz `RDEPEND` is empty by default in modern EAPIs. In older EAPIs, it was implicitly set to the value of `DEPEND` if not otherwise set in the ebuild.

Reply:
Made it clearer that this applies only if it's not empty.